bin/settings.py

```diff
@@ -12,6 +12,8 @@ def load_json(file_name):
 conf_data = load_json("creds.json")
 config = app.config
 
+config["SESSION_REFRESH_EACH_REQUEST"] = False
```

Comment:
what is this for?

Reply:
This config setting was mentioned in the Overview section of the ticket. After re-reading the Overview, I think this is unnecessary after the upgrade in Flask.

Reply:
ok, i'll remove it